demo/src/main/java/com/google/maps/android/utils/demo/CustomMarkerClusteringDemoActivity.java

```diff
@@ -123,7 +125,8 @@ protected void onClusterUpdated(Cluster<Person> cluster, Marker marker) {
          * @return a BitmapDescriptor representing a cluster
          */
         private BitmapDescriptor getClusterIcon(Cluster<Person> cluster) {
-            List<Drawable> profilePhotos = new ArrayList<Drawable>(Math.min(4, cluster.getSize()));
+            List<Drawable> profilePhotos;
+            profilePhotos = new ArrayList<Drawable>(Math.min(4, cluster.getSize()));
```

Comment:
I believe the newline was unintentional, and the `Drawable` type reference was meant to be removed?

```suggestion
List<Drawable> profilePhotos = new ArrayList<>(Math.min(4, cluster.getSize()));
```